### PR TITLE
Add some more logging during database upgrade checks

### PR DESCRIPTION
**Pull request by Commifreak**
This change adds some more informational log entries regarding database upgrade check.

I did not tested it. If something is wrong, I would adapt the patch as needed.

### CODE REVIEW

**Comment by uroni** [comment]
This would log a warning on every startup

**Comment by Commifreak** [comment]
Yes, that was intentional. Maybe the first warning should be info?

File: urbackupserver/dllmain.cpp [hunk 1]
```diff
@@ -2442,13 +2452,16 @@ void upgrade(void)
...
 		}
-		if(ver>max_v)
+		else if(ver>max_v)
 		{
 			Server->Log("Current UrBackup database version is "+convert(ver)+". This UrBackup"
 				" server version only supports databases up to version "+convert(max_v)+"."
 				" You need a newer UrBackup server version to work with this database.", LL_ERROR);
 			exit(4);
+		} else {
+		    Server->Log("No (more) database upgrade needed.", LL_WARNING);
```
Comment (uroni) [inline urbackupserver/dllmain.cpp:2462]: Can it get here?

**Comment by uroni** [comment]
Yes, or debug?

**Pull request closed without merging**